### PR TITLE
Fix refactor bug

### PR DESCRIPTION
**Pull request by PureW**
# Issue

What issue is this PR targeting? If there is no issue that addresses the problem, please open a corresponding issue and link it here.

## Tasklist

 - [ ] Add tests
 - [ ] Review - you must request approval to merge any PR to master
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Generally use squash merge to rebase and clean comments before merging
 - [ ] Update the [changelog](CHANGELOG.md)
 - [ ] Update relevant [documentation](docs/README.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: src/thor/timedep_forward.cc [hunk 1]
```diff
@@ -97,11 +97,11 @@ bool TimeDepForward::ExpandForward(GraphReader& graphreader,
       if (trans->up()) {
         hierarchy_limits_[node.level()].up_transition_count++;
         found_valid_edge = ExpandForward(graphreader, trans->endnode(), pred, pred_idx, true,
-                                         seconds_of_week, localtime, destination, best_path) ||
+                                         localtime, seconds_of_week, destination, best_path) ||
```
Comment (PureW) [inline src/thor/timedep_forward.cc]: This mixup of the order of `localtime` and `seconds_of_week` seems to have been in the code for over a year?

File: src/thor/timedep_forward.cc [hunk 2]
```diff
@@ -97,11 +97,11 @@ bool TimeDepForward::ExpandForward(GraphReader& graphreader,
...
                            found_valid_edge;
-      } else if (!hierarchy_limits_[trans->endnode().level()].StopExpanding()) {
+      } else if (!hierarchy_limits_[trans->endnode().level()].StopExpanding(pred.distance())) {
```
Comment (PureW) [inline src/thor/timedep_forward.cc]: I introducted this bug during the refactor in 6492d3dcc1f

File: src/thor/timedep_forward.cc (hunk 1 above)
Comment (dnesbitt61) [reply src/thor/timedep_forward.cc]: oops! Good catch. This code was never well tested (my only defense/excuse).

Comment (kevinkreiser) [reply src/thor/timedep_forward.cc]: i swear i fixed this myself a cuople months ago...

**Pull request merged**